RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -2163,38 +2116,37 @@ calculate_ele_mva(const pfEGHelpers::HeavyObjectCache* hoc,
       xtra.setGsfTrackPout(gsfElement->Pout());
       // normalization observables
       const float Pt_gsf = RefGSF->ptMode();
-      lnPt_gsf = std::log(Pt_gsf);
-      Eta_gsf = RefGSF->etaMode();
+      const float lnPt_gsf = std::log(Pt_gsf);
+      const float Eta_gsf = RefGSF->etaMode();
       // tracking observables
       const double ptModeErrorGsf = RefGSF->ptModeError();
-      dPtOverPt_gsf = (ptModeErrorGsf > 0. ? ptModeErrorGsf/Pt_gsf : 1.0);
-      nhit_gsf = RefGSF->hitPattern().trackerLayersWithMeasurement();
-      chi2_gsf = RefGSF->normalizedChi2();
-      DPtOverPt_gsf =  (Pt_gsf - gsfElement->Pout().pt())/Pt_gsf;
+      float dPtOverPt_gsf = (ptModeErrorGsf > 0. ? ptModeErrorGsf/Pt_gsf : 1.0);
+      //const float nhit_gsf = RefGSF->hitPattern().trackerLayersWithMeasurement();
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant